pkg/queue/cluster_queue_impl.go

```diff
@@ -221,6 +221,21 @@ func (c *clusterQueueBase) DumpInadmissible() (sets.Set[string], bool) {
 	return elements, true
 }
 
+func (c *clusterQueueBase) Snapshot(maxCount int32) ([]*kueue.Workload, bool) {
+	if c.heap.Len() == 0 {
+		return nil, false
+	}
+	elements := make([]*kueue.Workload, 0, c.heap.Len())
+	for i, e := range c.heap.List() {
```

Comment:
Please include `inadmissibleWorkloads` in the snapshot. Quoting the two reasons by @alculquicondor from a priv slack:
- usually all workloads will be in inadmissible, as kueue would be fast to process all jobs until the next job finishes.
- the inadmissible staging area is more of an implementation detail. It's just a mechanism for us to be able to look at the next elements in the queue.

Reply:
done